addons/google_gmail/models/ir_mail_server.py

```diff
@@ -27,7 +27,6 @@ def _onchange_use_google_gmail_service(self):
             self.smtp_encryption = 'starttls'
             self.smtp_port = 587
         else:
-            self.smtp_encryption = 'none'
```

Comment:
Would also move it in a specific commit updating gmail independently from outlook :)